.github/workflows/testing.yml

```diff
@@ -355,3 +355,48 @@ jobs:
       - name: Itemized validation
         if: ${{ !cancelled() }}
         run: bin/test_itemcheck
+
+  discovery:
+    name: Discovery Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    strategy:
```

Comment:
fail-fast not necessary b/c no matirx: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast